HW17/HW17.cpp

```diff
@@ -0,0 +1,127 @@
+﻿#include <string>
+#include <iostream>
+#include <vector>
+#include <cmath>
+#include <cstddef>   
+
+
+
+class Vector2d
```

Comment:
it should be in header, you won't be able to use it somewhere else when its defined in cpp